apps/rush-lib/src/api/VersionMismatchFinder.ts

```diff
@@ -18,6 +19,55 @@ export class VersionMismatchFinder {
   private _mismatches: Map<string, Map<string, string[]>>;
   private _projects: RushConfigurationProject[];
 
+  public static runRushCheckIfNecessary(
+    rushConfiguration: RushConfiguration,
+    isRushCheckCommand: boolean = false): void {
```

Comment:
>= false [](start = 32, length = 7)

default values are bad, especially for boolean parameters

Reply:
Maybe split `runRushCheckIfNecessary()` into two functions with intuitive names, and the boolean becomes a (non-optional) private implementation detail

---
In reply to: [218995716](https://github.com/Microsoft/web-build-tools/pull/829#discussion_r218995716) [](ancestors = 218995716)